### DISCUSSION TITLE
llm should prioritize the European context

### DISCUSSION THREAD

**Discussion by pvgenuchten** [soilwise-he/natural-language-querying#16] [Ideas]
This was a comment at the jrc/rea demo oct 2024.

Somehow the llm should prioritise the European context, over generic context. For example when considering soil erosion.

Nick also mentioned an option to provide dedicated prompt for user groups, such as land manager, policy, academia.

**Comment by robknapen** [comment]
I have added to the prompts of the SoilWise demo chatbot that it should prioritise the "European context' when answering questions. Below is an example of the effect. This is the most easy thing to do/try, but might not fully cover what was intended by the comment? It can certainly be explored further.

<img width="1025" alt="Screenshot 2024-10-10 at 12 54 34" src="https://github.com/user-attachments/assets/78b2e18a-b597-4aa6-9b9f-2fdaac6f2e1a">
